src/connection/auth.ts

```diff
@@ -270,15 +296,23 @@ class AccessTokenAuthenticator implements OidcAuthFlow {
 export interface ClientCredentialsInput {
   clientSecret: string;
   scopes?: any[];
+  silentRefresh?: boolean;
 }
 
-export class AuthClientCredentials {
+export class AuthClientCredentials implements OidcCredentials {
   private clientSecret: any;
   private scopes?: any[];
+  public readonly silentRefresh: boolean;
 
   constructor(creds: ClientCredentialsInput) {
     this.clientSecret = creds.clientSecret;
     this.scopes = creds.scopes;
+    // Silent token refresh by default
+    if (creds.silentRefresh === undefined) {
```

Comment:
just to be sure:
This means that this parameter is not required and this is not a breaking change?